### PR TITLE
Modified a lifecycle hooks documentation sentence to be more clear.

### PR DESCRIPTION
**Pull request by jordan-amman**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

The sentence isn't that readable.

Issue Number: N/A

## What is the new behavior?
The sentence is more readable and easier to understand.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x ] No

## Other information
@kapunahelewong, this is the pull request you suggested, thank you for your help! [:

### CODE REVIEW

**Comment by jordan-amman** [comment]
@kapunahelewong Just wanted to make sure you saw this pull request since you suggested it. Thanks again!

File: aio/content/guide/lifecycle-hooks.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 A component has a lifecycle managed by Angular.
 
-Angular creates it, renders it, creates and renders its children,
-checks it when its data-bound properties change, and destroys it before removing it from the DOM.
+Angular creates and renders components and their children, checks when their data-bound properties change, and destroys them before removing them from the DOM.
```
Comment (kapunahelewong) [inline aio/content/guide/lifecycle-hooks.md]: How about this little change? (I stumbled on the second "and" when I first read it.)

```suggestion
Angular creates and renders components along with their children, checks when their data-bound properties change, and destroys them before removing them from the DOM.
```

**Review by kapunahelewong — CHANGES_REQUESTED** [review]
Thanks for this, @jordan-amman! Just one small thing. 

Will you tag me when you're ready?

EDIT: Oh, you'll need to amend the commit message so it'll pass, too. Try something like:

```
docs: clarify sentence in lifecycle hooks doc
```

[Here](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-amend-a-commit) are my notes on how to amend commit messages if you'd find some helpful. :)

File: aio/content/guide/lifecycle-hooks.md (hunk 1 above)
Comment (jordan-amman) [reply aio/content/guide/lifecycle-hooks.md]: So I added these changes to a new commit, but it also shows the old commit, not sure what I'm supposed to do about that? Thanks again for your help!

Comment (kapunahelewong) [reply aio/content/guide/lifecycle-hooks.md]: Ah, check out these steps. You just need to squash. [Here's what I do](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-squash).

Let me know if you have any questions. :)

**Pull request closed without merging**